documentation/build.gradle

```diff
@@ -98,8 +98,16 @@ task expandJavaDocIndexHtml(type: Copy) {
 	)
 }
 
+task generateUserGuidePDF(type : Exec) {
+	dependsOn generateUserGuideToc
+
+	workingDir "$buildDir/docs"
+
+	commandLine 'pandoc', '--pdf-engine=xelatex', '-s', '-o', 'user-guide.pdf', 'user-guide.md'
```

Comment:
Should this specify `task inputs` and `task outputs`?
Inputs and outputs would enable Gradle to track up-to-date properly, and the task could even become build-cacheable in the future.